### DISCUSSION TITLE
Possible to add custom model to admin menu?

### DISCUSSION THREAD

**Discussion by johnrisby** [swellstores/community#373] [Help] [answered]
From reading the docs, it seemed to suggest it's possible to add a custom content model - but not a custom data model - to the admin backend, but something else I've just read in the docs seems to say content can only be added/edited etc via the backend API?

Is there any way to add records to custom models via the admin ui, and add a page link to the menu etc? Thanks.

**Comment by ericingram** [comment, marked as answer]
It's one of the main features of Apps

**Reply by johnrisby** [reply]
Oh, stop teasing me with Apps! ;)

**Reply by JoeMatkin** [reply]
I had the same question, how is this done?

What is meant by "it's one of the main features of apps"?

**Reply by ericingram** [reply]
By that I mean, you will be able to create an app that can add custom models to the left nav, with custom views to manage content, and also add things to standard areas including tabs in lists and in all record views (order, products, etc). Apps should make managing customizations much easier while also expanding the ways an app can enhance the dashboard.

**Reply by ericingram** [reply]
I should note, Apps are in private alpha stage right now and we will get to open beta some time in the next few months.
